src/types/types.ts

```diff
@@ -9,3 +9,18 @@ export interface ISignUpPayload {
   email: string;
   password: string;
 }
+
+export interface IAttributesInput {
```

Comment:
This doesn't seem right. Use string type. Or instead of creating an interface in another file for components props, create the type Props in the same component file before declaring the  function component.
Eg:
`type Props {
  id: string;
  name: string
  type: string (or  'text' | 'number' etc.)
  placeholder: string
}`
Or extend the Filed props.